doc/Reactive.md

```diff
@@ -16,42 +70,285 @@ Each [`Message<T>`](https://github.com/unoplatform/uno.extensions/blob/main/src/
+}
+
+public record WeatherInfo(double Temperature, WeatherAlert? Alert);
+
+public record WeatherAlert(Guid Id, string? Title);
+
+public interface ILocationService
+{
+	/// <summary>
+	/// Asynchronously gets the name of the current city.
+	/// </summary>
+	Task<string> GetCurrentCity(CancellationToken ct);
+
+	/// <summary>
+	/// Gets teh list of all supported cities.
+	/// </summary>
+	Task<ImmutableList<string>> GetCities(CancellationToken ct);
+}
+```
+
+### Async
+Creates a feed from an async method. The loaded data can be refreshed using a `Signal` trigger that will re-invoke the async method.
+
+```csharp
+private IWeatherService _weatherService;
+
+public IFeed<WeatherInfo> Weather => Feed.Async(async ct => await _weatherService.GetCurrentWeather(ct));
+```
+
+### AsyncEnumerable
+This adapts an `IAsyncEnumerable<T>` into a _feed_
+
+```csharp
+private IWeatherService _weatherService;
+
+public IFeed<WeatherInfo> Weather => Feed.AsyncEnumerable(() => GetWeather());
+
+private async IAsyncEnumerable<WeatherInfo> GetWeather([EnumeratorCancellation] CancellationToken ct = default)
+{
+	while (!ct.IsCancellationRequested)
+	{
+		yield return await _weatherService.GetCurrentWeather(ct);
+		await Task.Delay(TimeSpan.FromHours(1), ct);
+	}
+}
+```
+
+### Create
+This gives you the ability to create your own _feed_ by dealing directly with _messages_.
+
+> This is designed for advanced usage and should probably not be used directly in apps.
+
+```csharp
+public IFeed<WeatherInfo> Weather => Feed.Create(GetWeather);
+
+private async IAsyncEnumerable<Message<WeatherInfo>> GetWeather([EnumeratorCancellation] CancellationToken ct = default)
+{
+	var message = Message<WeatherInfo>.Initial;
+	var weather = Option<WeatherInfo>.Undefined();
+	var error = default(Exception);
+	while (!ct.IsCancellationRequested)
+	{
+		try
+		{
+			weather = await _weatherService.GetCurrentWeather(ct);
+			error = default;
+		}
+		catch (Exception ex)
+		{
+			error = ex;
+		}
+
+		yield return message = message.With().Data(weather).Error(error);
+		await Task.Delay(TimeSpan.FromHours(1), ct);
+	}
+}
+```
+
+## Operators: How to interact with a feed
+You can apply some operators directly on any _feed_.
+
+> Note : You can use the linq syntax with feeds:
+> ```csharp
+> public IFeed<string> Value => from value in _values
+> 	where value == 42
+> 	select value.ToString();
+> ```
+
+### Where
+Applies a predicate on the _data_.
+
+Be aware that unlike `IEnumerable`, `IObservable` and `IAsyncEnumerable`, if the predicate returns false, a message with a `None` _data_ will be published.
+
+```csharp
+public IFeed<WeatherAlert> Alert => Weather
+	.Where(weather => weather.Alert is not null)
+	.Select(weather => weather.Alert!);
+```
+
+### Select
+Synchronously projects each data from the source feed.
+
+```csharp
+public IFeed<WeatherAlert> Alert => Weather
+	.Where(weather => weather.Alert is not null)
+	.Select(weather => weather.Alert!);
+```
```

Comment:
```suggestion
```csharp
public IFeed<WeatherInfo> Alert => Weather
	.Where(weather => weather.Alert is not null);
```
```
Remove projection since this is shown in the next csharp block

Reply:
A would agree, but I was not able to find an example that really make sense without the projection :/